packages/itwinui-css/src/tabs/pill.scss

```diff
@@ -1,34 +1,43 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import './tabs';
 
 @mixin iui-tabs-pill {
   justify-content: space-evenly;
-  gap: $iui-xxs;
+  gap: var(--iui-size-3xs);
 
   > li {
     flex: 1 1 0;
   }
 
-  &.iui-green .iui-tab:hover {
-    background-color: rgba(var(--iui-color-foreground-positive-rgb), var(--iui-opacity-6));
+  &.iui-green .iui-tab {
+    &:hover {
+      background-color: var(--iui-color-background-transparent-hover);
+    }
+
+    &.iui-active:hover {
+      background-color: var(--iui-color-background-subtle-positive);
+    }
   }
 
   .iui-tab {
-    padding: $iui-xs;
+    padding: var(--iui-size-2xs);
     background-color: transparent;
     width: 100%;
     justify-content: center;
-    border-radius: $iui-border-radius;
+    border-radius: var(--iui-border-radius-1);
 
     &::after {
       bottom: 0;
       left: 0;
     }
 
     &:hover {
-      background-color: rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6));
+      background-color: var(--iui-color-background-transparent-hover);
+    }
+
+    &.iui-active:hover {
```

Comment:
It doesn't seem to work on regular tab. But I am more of a fan of having colorful hover states as before.

Reply:
what doesn't work? this code is for pill tabs

Reply:
![tab-v1-hover](https://user-images.githubusercontent.com/36186912/187159242-0da05b8e-7dd7-4ada-899e-0e7f72eba72f.gif)

Reply:
Fixed